spec/javascripts/LoginPage/LoginPageWrapper.spec.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Enzyme, {mount} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import {SimpleLoginPage, Login3scaleForm, RequestPasswordForm} from 'LoginPage'
+
+Enzyme.configure({adapter: new Adapter()})
+
+const props = {
+  enforceSSO: false,
+  authenticationProviders: null,
+  providerAdminDashboardPath: 'provider-admin-path',
+  providerLoginPath: 'provider-login-path',
+  providerPasswordPath: 'password-path',
+  providerSessionsPath: 'sessions-path',
+  redirectUrl: 'redirect-url',
+  show3scaleLoginForm: true
+}
+
+it('should render itself with right props', () => {
+  const wrapper = mount(<SimpleLoginPage {...props}/>)
+  expect(wrapper.find('.pf-c-login').exists()).toEqual(true)
+  expect(wrapper.props().authenticationProviders).toEqual(null)
+  expect(wrapper.props().providerAdminDashboardPath).toEqual('provider-admin-path')
+  expect(wrapper.props().providerLoginPath).toEqual('provider-login-path')
+  expect(wrapper.props().providerPasswordPath).toEqual('password-path')
+  expect(wrapper.props().providerSessionsPath).toEqual('sessions-path')
+  expect(wrapper.props().redirectUrl).toEqual('redirect-url')
+  expect(wrapper.props().show3scaleLoginForm).toEqual(true)
+})
+
+it('should call getURL method on mount', () => {
```

Comment:
if we changed this method and adopt also DI for the window object, you could assert the functionality of changing the state of the form with different href values